modules/op-release-notes-1-16.adoc

```diff
@@ -0,0 +1,615 @@
+
+* Before this update, when you used one of the `s2i-*` tasks defined in the {pipelines-shortname} installation namespace and set its `CONTEXT` parameter, the task did not interpret the parameter correctly and the task failed. With this update, the task interprets the `CONTEXT` parameter correctly and completes successfully.
+
+* Before this update, in {tekton-chains} the in-toto provenance metadata, `URI` and `Digest` values, were incomplete. The values contained only the information of remote `Pipeline` and `Task` resources, but were missing the information of the remote `StepAction` resource. With this update, the provenance of the remote `StepAction` resource is recorded in the task run status and inserted into the in-toto provenance, which results in complete in-toto provenance metadata.
+
+* Before this update, you could modify some of the parameters in the `spec` field of the `PipelineRun` and `TaskRun` resources that should not be modifiable after the resources were created. With this update, you can only modify the allowed fields after the pipeline run and task run are created, such as `status` and `statusMessage` fields.
+
+* Before this update, if a step action parameter was an `array` type but a `string` value was passed in a task, there was no error indicating inconsistent parameter types and the default parameter value was used instead. With this update, an error is added to indicate the inconsistent values: `invalid parameter substitution: %s. Please check the types of the default value and the passed value`.
+
+* Before this update, task runs and pipeline runs were deleted by an external pruner when logs were streamed through the watcher. With this update, a finalizer is added to {tekton-results} for `TaskRun` and `PipelineRun` objects to ensure that the runs are stored and not deleted. The runs are stored either as records or until the deadline has passed, which is calculated as the completion time plus the `store_deadline` time. The finalizer does not prevent deletion if legacy log streaming from the watcher or pruner is enabled. 
+
+* Before this update, the web console supported the `v1beta1` API format to display the `TaskRun` and `PipelineRun` object records that are stored in {tekton-results}. With this update, the console supports the `v1` API format to display `TaskRun` and `PipelineRun` object records stored in {tekton-results}. 
+
+* Before this update, when using {pac}, if different `PipelineRun` definitions used the same task name but different versions, for example when fetching tasks from {tekton-hub},the wrong version was sometimes triggered, because {pac} used the same task version for all pipeline runs. With this update, {pac} triggers the correct version of the referenced task.
```

Comment:
Missing space:
```suggestion
* Before this update, when using {pac}, if different `PipelineRun` definitions used the same task name but different versions, for example when fetching tasks from {tekton-hub}, the wrong version was sometimes triggered, because {pac} used the same task version for all pipeline runs. With this update, {pac} triggers the correct version of the referenced task.
```